packages/components/CHANGELOG.md

```diff
@@ -204,6 +204,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 + <p-icon name="arrow-right"></p-icon>
 ```
 
+#### Added
+
+- `Carousel` supports new prop `alignHeader`:
+
+```diff
++ <p-carousel alignHeader="center"></p-carousel>
```

Comment:
No need for a diff since we don't need a migration path or anything.

Reply:
Fixed